modules/react-native-wallet/src/ApplePushProvisioning.ts

```diff
@@ -0,0 +1,89 @@
+import {NativeEventEmitter, NativeModules} from 'react-native';
+import type {NativeModule} from 'react-native';
+
+type AddPassRequest = {
+    last4: string;
+    cardHolderName: string;
+};
+
+type CompletePassRequest = {
+    activation: string;
+    encryptedData: string;
+    ephemeralKey: string;
+};
+
+type GetPassAndActivationEvent = {
+    certificateLeaf: string;
+    certificateSubCA: string;
+    nonce: string;
+    nonceSignature: string;
+};
+
+type SupportedEvents = 'addPaymentPassViewControllerDidFinish' | 'getPassAndActivation';
+
+type ApplePushProvisioningType = NativeModule & {
+    canAddPaymentPass: () => Promise<boolean>;
+    startAddPaymentPass: (last4: string, cardHolderName: string) => Promise<void>;
+    completeAddPaymentPass: (activation: string, encryptedData: string, ephemeralKey: string) => Promise<void>;
+};
+
+const ApplePushProvisioning = (NativeModules.ApplePushProvisioning as ApplePushProvisioningType) ?? null;
+if (!ApplePushProvisioning) {
```

Comment:
Still react native tries to load Google and Android modules at once, which result in error
```suggestion
if (Platform.OS === 'ios' && !ApplePushProvisioning) {
```